FindTeamer/ViewsContent/MainContent/MainContentView.swift

```diff
@@ -11,6 +11,7 @@ struct MainContentView: View {
     // MARK: - Private properties
     @State private var presentAddEventSheet = false
     @ObservedObject private var viewModel: EventsViewModel
+    private let logInViewModel: LogInViewModel = LogInViewModel(authManager: FirebaseAuthService())
```

Comment:
Here when you pass it a new object will be created each time. There should be a single object that's injected, otherwise you would have different objects in all the places where you pass the AuthService.